docs/components/text.md

```diff
@@ -0,0 +1,91 @@
+---
+title: text
+type: components
+layout: docs
+parent_section: components
+---
+
+The text component renders bitmap and signed distance field font text.
+
+From Fernando Serrano's aframe-bmfont-component with further modifications by Michael Chen,
```

Comment:
I believe we should remove our names from here, in a readme as it was originally written makes sense to credit the sources and so on but here we're writing end-user documentation and I don't think it's useful anymore that info.